packages/components/psammead-brand/package.json

```diff
@@ -24,7 +24,9 @@
   },
   "devDependencies": {
     "@bbc/psammead-script-link": "^3.0.14",
-    "@bbc/psammead-styles": "^7.2.2"
+    "@bbc/psammead-styles": "^7.2.2",
+    "@emotion/styled": "^11.0.0",
+    "react": "^17.0.1"
```

Comment:
how do these dev dependencies work with the peer dependencies here? I guess the peer dependencies declare that you can integrate and use this package with a project (Simorgh in our case) that uses Emotion styled > 11 and react > 16.9 ? Should we change the react one to be react 17? I imagine this will be needed across all packages

Reply:
I _think_ if we do then we introduce a backwards-incompatible breaking change because consumers who haven't upgraded to React v17 won't be able to use new versions of Psammead components.